tutorials/editor/script_editor.rst

```diff
@@ -0,0 +1,409 @@
+- **Theme**: Provides options to import an existing theme, save or reload it. Changing 
+  theme settings is performed via `Editor Settings`.
+- **Close**: Closes the active script.
+- **Close All**: Closes all opened scripts and asks to save if there are unsaved changes.
+- **Close Other Tabs**: Closes all opened scripts except the selected one.
+- **Close Docs**: Closes the class reference documentation pages, leaving only the
+  scripts opened.
+- **Run**: If the script extends the :ref:`EditorScript <class_EditorScript>` and
+  intended to be executed without running the project, this option runs the script.
+  See :ref:`doc_running_code_in_the_editor_editorscript` for more.
+- **Toggle Scripts Panel**: Shows or hides the script panel located on the left side
+  of the text editor, allowing you to expand the available coding area. More on the
+  `Scripts Panel` will be explained below.
+
+**Edit** menu several provides options for line operations:
+
+.. image:: img/script_editor_icons/text_editor_edit_menu.webp
+
+- **Undo***: Allows you to reverse the most recent action or series of actions, restoring
+  document or code to its previous state before the changes were made.
+- **Redo***: Allows you to reapply an action that was previously undone, effectively
+  redoing the last action that was reversed by the Undo function.
+- **Cut***: Cuts the selection into the clipboard.
+- **Copy***: Copies the code selection into the clipboard.
+- **Paste***: Pastes the content of the clipboard if it contains text.
+- **Select All***: Selects the all code in the text editor.
+- **Duplicate Selection**: Copies the selection and appends it next to the selection.
+- **Duplicate Lines**: Duplicates the current line and adds it as a new line below the
+  current line.
+- **Evaluate Selection***: Computes the values of the selected text if it contains `only`
+  a mathematical expression, such as ``83 * 3`` or ``pow(2,3)``.
+- **Toggle Word Wrap**: Disables the horizontal scrollbar by wrapping the long lines to
+  the next line. Note that this is just a visual change and no new linebreaks are added.
+- **Line**: Provides a set of line operations. Depending on the opened file, the options
+  might also be directly in the Edit menu, instead of a submenu.
+
+  - `Move Up`: Moves the current line or the selected line(s) one line up.
+  - `Move Down`: Moves the current line or the selected line(s) one line down.
+  - `Indent*`: Indents the text from the caret or the selected line(s), following the
+    indentation setting.
+  - `Unindent*`: Unindents the text from the caret or the selected line(s), following the
+    indentation setting.
+  - `Delete Line`: Deletes the current line or the selected line(s).
+  - `Toggle Comment*`: Comments and uncomments the current line or the selected line(s).
+    You can perform the same action by selecting line(s) and choosing the same action
+    after right-clicking on the selected text.
```

Comment:
The items here are not the context menu items, but the main menu. Since it is the first time I am mentioning this "Toggle Comment", I thought that it make sense to keep it.

Reply:
@tetrapod00 What do you think about this? Can we resolve this?

Reply:
I ended up merging this as-is.